### ISSUE TITLE
Absolute instead of relative path for image upload?

### ISSUE THREAD

**Issue by wwsalmon** [Ionaru/easy-markdown-editor#305]
I'm implementing inlnie image upload and have an endpoint set up to upload the image to an S3 bucket and return the URL.

However, the URL is treated as a relative path, i.e. the inserted image is `![](http://localhost:3000/https://postulate.s3.us-west-1.amazonaws.com/myimage.jpg)`. Any way to have the returned path treated as an absolute URL?

**Comment by wwsalmon** [comment]
Looking through the source code, looks like path is necessarily relative (easymde.js line 2343):

```
onSuccess(window.location.origin + '/' + response.data.filePath);
```

Any chance of adding an option like boolean `imagePathAbsolute` to support absolute paths? I'd be willing to make a PR if it can be merged soon!

**Comment by Ionaru** [comment]
Closed in #313
